rotate.py

```diff
@@ -0,0 +1,26 @@
+import cv2 as cv

```

Comment:
Unless I overlooked something, this file is not used by any of the other files, and doesn't have a "productive" function by itself.
If we should decide to introduce the file structure that I suggested in the main review comment, how about putting this file in `src/examples` or maybe `src/experiments`?

Reply:
Yes this file was just an example. From my point of view it was intended as an example and now can be completely deleted from the repository.

Reply:
As you wish, for me both options are fine: either deleting it completely or putting it in a dedicated examples folder